docs/experiments-msmarco-v2.md

```diff
@@ -49,9 +49,9 @@ Evaluation:
 
 ```bash
 $ tools/eval/trec_eval.9.0.4/trec_eval -c -m map -m recall.100,1000 -m recip_rank collections/passv2_dev_qrels.uniq.tsv runs/run.msmarco-pass-v2.dev.txt
-map                   	all	0.0709
-recip_rank            	all	0.0719
-recall_100            	all	0.3397
+map                     all     0.0718
```

Comment:
do we also wanna just also include the top100 rerank results?

Reply:
As discussed let's go with the top 100 itself?